src/pyhf/pdf.py

```diff
@@ -231,21 +231,27 @@ def _dataprojection(self, data):
         return data[cut:]
 
     def has_pdf(self):
-        '''
+        """
+        Indicate whether the model has a constraint.
+
         Returns:
             flag (`bool`): Whether the model has a constraint term
-        '''
+        
```

Comment:
Also following up on docstring style, do we want

```
Returns:
    <information on the return>
"""
```

or with the extra space

```
Returns:
    <information on the return>

"""
```

?

Reply:
pydocstyle requires a newline